__mocks__/@keg-hub/git-lib.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
This should be part of the git-lib repo, that way it does not have to be duplicated for every app the uses it. Then you can just import it directly when mocking the git-lib

Reply:
I actually determined I didn't need these mocks. `jest.mock` can already automock a package's whole api, so I udpated this to just do that.